csrc/utils.h

```diff
@@ -10,6 +10,8 @@
 #include <ATen/ATen.h>
 #include <c10/util/Exception.h>
 #include <torch/csrc/jit/ir/ir.h>
+
+#include <options.h>
```

Comment:
nit: instead of making `utils.h` include `options.h`, should we modify every file using options and let it include `options.h` directly?